test/puppetlabs/ssl_utils/testutils.clj

```diff
@@ -188,12 +188,12 @@
             crls [root-crl]
             certs-to-generate (dec number-of-certs)
             issuer root-cert
-            issuer-key-pair root-key-pair]
+            previous-key-pair root-key-pair]
```

Comment:
Is there a reason you renamed this? I feel like `issuer` was a little clearer to me.

Reply:
It was helpful to me as I was working on this, so it was clear that this is the key from the previous iteration, but I can put it back.

Reply:
okay updated